manifests/repo/yum.pp

```diff
@@ -8,12 +8,16 @@
     if $sensuclassic::repo_source {
       $url = $sensuclassic::repo_source
     } else {
+      $major = $facts.dig('os', 'release', 'major')
```

Comment:
why dig? why not just address the facts by name?

Reply:
Because it would require more complete unit tests because if you do `$facts['os']['family']` and `$facts['os']` is not defined, you get errors.  Dig is much safer when traversing deep hash in case facts are missing.

Reply:
Also ever since Puppet 5 dig is the better way to walk a hash, just safer and less verbose than checking each intermediate key exists.  Might have been around since Puppet 4, but I know it's built-in to Puppet 5.